strawberryfields/engine.py

```diff
@@ -359,7 +350,11 @@ def _run_program(self, prog, **kwargs):
                         cmd.op, self.backend, kwargs
                     )
                 ) from None
-        return applied
+
+        if values is None:
+            values = np.array([[]])
+
+        return applied, values
```

Comment:
Is the returned `applied` never used? 🤔

Reply:
It doesn't look like it is. :thinking: I guess it could be removed then?

Reply:
just curious if `applied` should eventually be removed

Reply:
Some tests seem to be using it at the moment, so I've left it for now. Perhaps this could be looked at in another PR?